src/api/getSecurityQuestion.js

```diff
@@ -14,20 +13,14 @@ router.post("/getSecurityQuestion", check("email").isEmail(), async function(
       errors: errors.array({ onlyFirstError: true })
     });
   }
-
-  const user = await User.findOne({ email: req.body.email });
-  if (!user) {
-    sendResponse(res, 400, "User does not exist in the database");
-    return;
-  }
-  if (!user.question) {
-    sendResponse(res, 400, "No security question set");
-    return;
+  const questionsResponse = await getSecurityQuestions();
+  if (!questionsResponse.success) {
+    return sendResponse(res, 500, "something went wrong on our end");
```

Comment:
Instead, the error message should be "No security question could be parsed from the config file"